blockonomics-woocommerce.php

```diff
@@ -223,122 +218,232 @@ function show_basic() {
-                        </th>
-                        <td><?php echo get_callback_url(); ?></td>
-                    </tr>
-                    <tr valign="top">
-                        <th scope="row"><?php echo __('BCH Enabled (Select if you want to accept Bitcoin Cash)')?></th>
-                        <td><input type="checkbox" name="blockonomics_bch" value="1" <?php checked("1", get_option('blockonomics_bch')); ?>" /></td>
-                    </tr>
-                    <tr valign="top">
-                        <th scope="row">Destination BTC wallet for payments</th>
-                        <td>
-                            <?php
-                            $total_received = get_option('blockonomics_temp_withdraw_amount') / 1.0e8;
-                            $api_key = get_option("blockonomics_api_key");
-                            $temp_api_key = get_option("blockonomics_temp_api_key");
-                            if ($temp_api_key && !$api_key && !($total_received > 0)): ?>
-
-                            <p><b>Blockonomics Wallet</b> (Balance: 0 BTC)</p>
-                            <p>We are using a temporary wallet on Blockonomics to receive your payments.</p>
-                            <p>To receive payments directly to your wallet (recommended) -> Follow Wizard by clicking on <i>Get Started for Free</i> on <a href="https://www.blockonomics.co/merchants" target="_blank">Merchants</a> and enter the APIKey below [<a href="https://blog.blockonomics.co/how-to-accept-bitcoin-payments-on-woocommerce-using-blockonomics-f18661819a62">Blog Instructions</a>]</p>
-
-                            <?php elseif ($temp_api_key && $total_received > 0): ?>
-
-                            <p><b>Blockonomics Wallet</b> (Balance: <?php echo "$total_received"; ?> BTC)</p>
-                            <?php if (!$api_key): ?>
-                            <p> To withdraw, follow wizard by clicking on <i>Get Started for Free</i> on <a href="https://www.blockonomics.co/merchants" target="_blank">Merchants</a>, then enter the APIKey below [<a href="https://blog.blockonomics.co/how-to-accept-bitcoin-payments-on-woocommerce-using-blockonomics-f18661819a62">Blog Instructions</a>]
-                            </p>
-                            <?php else: ?>
-                            <p> To withdraw, Click on <b>Test Setup</b></p>
+                <?php wp_nonce_field('update-options');
+                switch ( $active_tab ){
+                case 'settings' :
+                    ?>
+                    
+                    <h1>API Key</h1>
+                    <input onchange="value_changed()" size="130" type="text" id="blockonomics_api_key" name="blockonomics_api_key" value="<?php echo get_option('blockonomics_api_key'); ?>" />
```

Comment:
The inputs are not responsive on smaller screens. Add `max-width: 100%;` to fix and `text-overflow: ellipsis;` to indicate if text is longer than the input.

Reply:
Done, made the input fields responsive.